### ISSUE TITLE
ShedLock should not pull junit-vintage-engine

### ISSUE THREAD

**Issue by mickaeltr** [lukas-krecan/ShedLock#179]
Hello,

ShedLock 4.0.2 uses `junit-vintage-engine` but forgot to declare it as a **test** dependency.

This is my workaround for now:

```xml
    <dependency>
      <groupId>net.javacrumbs.shedlock</groupId>
      <artifactId>shedlock-spring</artifactId>
      <version>${shedlock-spring.version}</version>
      <exclusions>
        <exclusion>
          <groupId>org.junit.vintage</groupId>
          <artifactId>junit-vintage-engine</artifactId>
        </exclusion>
      </exclusions>
    </dependency>

    <dependency>
      <groupId>net.javacrumbs.shedlock</groupId>
      <artifactId>shedlock-provider-jdbc-template</artifactId>
      <version>${shedlock-spring.version}</version>
      <exclusions>
        <exclusion>
          <groupId>org.junit.vintage</groupId>
          <artifactId>junit-vintage-engine</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

Thanks

**Comment by lukas-krecan** [comment]
Good catch, thanks

**Comment by lukas-krecan** [comment]
Released as 4.0.3, thanks again
